CHANGELOG.md

```diff
@@ -5,12 +5,32 @@ Notable changes to this project are documented in this file. The format is based
 ## [Unreleased]
 
 Breaking changes:
+- Added support for PureScript 0.14 and dropped support for all previous versions (#181)
+- Renamed `Data.Array.ST.empty` to `Data.Array.ST.new` (#191, #198)
+- Renamed `group'` to `groupAll` (#194, #200)
 
 New features:
+- Added specialized monomorphic functions from `Data.Foldable` (#201):
```

Comment:
I actually find this a bit confusing, because these functions aren’t actually from Data.Foldable, they’re versions of those functions. I’d suggest “Added specialized versions of the functions from Data.Foldable”. Saying they’re specialized is the same thing as saying they’re monomorphic, and I think using both words makes this a bit harder to parse.

Reply:
What about
> Exported monomorphic functions from `Data.Foldable` (i.e. `f` is forced to be `Array`)

It doesn't include the `HeytingAlgebra` being forced to be `Boolean` changes we also made (e.g. `all`, `any`), but that's the heart of it.

Reply:
I still prefer to say “Added specialized versions of the functions from Data.Foldable“. My problem is with saying the functions are from Data.Foldable (as opposed to being versions of the functions from Data.Foldable).